### ISSUE TITLE
[bug] Support Sepolia chainID

### ISSUE THREAD

**Issue by Elliott-Green** [MetaMask/etherscan-link#72]
GIVEN I pass a chainID of 0x11155111 (Sepolia)
WHEN I query createExplorerLinkForChain() with the Sepolia chainID
THEN I expect the string `https://sepolia.etherscan.io/tx/` to be returned

BUT it returns empty string. 

```ts
<script lang="ts">
    import { createExplorerLinkForChain} from '@metamask/etherscan-link';
</script>

createExplorerLinkForChain('0xtxhashgoeshere', '0x11155111')
```

**Comment by mcmire** [comment]
Hey @Elliott-Green, thanks for the report. It seems support for Sepolia was added in #64, but we haven't yet issued a new release with these changes. I'll leave this issue open until we've done that.

**Comment by Elliott-Green** [comment]
@mcmire Any chance of a new release soon?

**Comment by mcmire** [comment]
Hey @Elliott-Green, so sorry about the delay and thanks for the reminder. This is now released under 3.0.0. Cheers.
